doc/js-api/remotestorage.rst

```diff
@@ -12,46 +15,18 @@ The constructor can optionally be called with a configuration object, for
 example::
 
    var remoteStorage = new RemoteStorage({
-     logging: true,  // defaults to false
-     cordovaRedirectUri: 'https://app.wow-much-app.com' // defaults to undefined
-   });
-
-Here is a complete list of the default options (set in ``src/config/js``), all
-of which can be overridden by calling the constructor with a configuration
-object like above::
-
-   {
-     // Enable remoteStorage logging
-     logging: false,
-     // Change Events that are enabled. See the BaseClient API for more information
+     cache: true,
```

Comment:
Where did the comments with the defaults go?

Reply:
I moved them to the constructor's argument docs, do you think it's better to have them both here and there?